markdown.go

```diff
@@ -393,7 +393,6 @@ func firstPass(p *parser, input []byte) []byte {
 		tabSize = TAB_SIZE_EIGHT
```

Comment:
5 lines above, the comment for `firstPass` func still says:

```Go
// - add missing newlines before fenced code blocks
```

That should be removed.